AAInstrumentation/core.py

```diff
@@ -236,6 +236,23 @@ def run_and_assemble_file(
             / Path(str(name_prefix) + str(file_name.stem) + ".bc")
         )
 
+    def run_assemble_and_measure_file(
+        self,
+        file_name: Path,
+        name_prefix: int,
+        index_list: list[int],
+        instrument_recursively=False,
+    ) -> int:
+        """Run a single function, assemble the result, and measure the output size."""
```

Comment:
> Run a single function

what does this mean?

Reply:
It means I copy pasted this from the old instrumentation. Change to "Run a round of instrumentation, assemble the result, and measure the output size.